source/intro.tex

```diff
@@ -359,20 +349,6 @@
 \definition{implementation limits}{defns.impl.limits}
 restrictions imposed upon programs by the implementation
 
-\definition{iostream class templates}{defns.iostream.templates}
```

Comment:
This is referred to in a note in [defns.character.container]:

"It is used for one of the template parameters of the string, iostream, and regular expression class
templates."

Reply:
#6236 will also be fixed once [defns.iostream.templates] is correctly removed by this PR.

Reply:
Ah yes, I knew I'd discussed this defn recently. I still think my suggestion there would improve [defns.character.container].

Reply:
We'll also get to your suggestion!